scripts/git/release.sh

```diff
@@ -3,16 +3,30 @@
 # Note: Windows Bash doesn't support shebang extra params
 set -e
 
-VERSION=$1
-ACCESS_TOKEN=$2
-DOCKER_USER=$3
-DOCKER_PWD=$4
-FROM_DOCKERHUB_NAMESPACE=${5:-azureiotpcs}
-TO_DOCKERHUB_NAMESPACE=${6:-azureiotpcs}
-SOURCE_TAG="${7:-testing}"
-DESCRIPTION=$8
-PRE_RELEASE=${9:-false}
-LOCAL=${10}
+while [ "$#" -gt 0 ]; do
```

Comment:
Were you able to run a test release using this script?

Reply:
partial test, not a completed test yet since it will push tags on github and docker hub.